net/netutil/ip_forward.go

```diff
@@ -212,9 +212,11 @@ func ipForwardingEnabledLinux(p protocol, iface string) (bool, error) {
 		}
 		return false, err
 	}
-	on, err := strconv.ParseBool(string(bytes.TrimSpace(bs)))
+
+	val, err := strconv.ParseInt(string(bytes.TrimSpace(bs)), 10, 32)
 	if err != nil {
 		return false, fmt.Errorf("couldn't parse %s: %w", k, err)
 	}
+	on := val == 1 || val == 2
```

Comment:
should you return an error on negative values or val greater than 2?

Reply:
Updated, PTAL!